docs-2.0/3.ngql-guide/3.data-types/6.list.md

```diff
@@ -0,0 +1,12 @@
+# List
+
+List is a composite data type. A composite data type cannot be stored as properties. List is a sequence of values. Individual list elements can be accessed by their positions.
+
+A list starts with a left square bracket `[` and ends with a right square bracket `]`. List contains zero, one or more expressions. List elements are separated from each other with comma `,`. Whitespace around elements is ignored in list, thus line breaks, tab stops and blanks can be used for formatting.
```

Comment:
List contains zero, one, or more expressions. Add a comma.
List elements are separated from each other with a comma. Add "a".
tab stops, and blanks. Add a comma.

Reply:
one or more doesn't need a comma I think

Reply:
It's "zero, one, or more". Or maybe "zero to multiple".

Reply:
ah I see